### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by sidvishnoi**
Using the [new issue template workflow](https://help.github.com/articles/about-issue-and-pull-request-templates)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- URL to affected spec:
+- ReSpec version: Click the ReSpec pill in top-right corner (e.g., 25.12.0).
+- [x] I did a "hard refresh", but it's still busted.
+
+## Description of problem
+
+**What happened (e.g., it crashed)?:**
+
+**But what I was expecting?:**
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
**Expected behavior (e.g., it shouldn't crash):**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]:
```suggestion
**Can you or your organization fund the work or help with development?**
ReSpec is a community maintained project. To add a new feature, we usually require funding to pay for feature development and server costs. 
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ""
+labels: Feature request
+assignees: ""
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
A clear and concise description of what the problem is. E.g. "I'm always frustrated when [...]"
```

**Comment by ONFU** [comment]
the live message will be follow of districk NoX-PacT the riddick haved changing by chacal of disturb NAuthority and authority his my sons and his my general administratif a lnternational of worl wars and NoX-PacT and his killer of etat united stade of America

**Comment by ONFU** [comment]
live message with follow lt's the reddick ouped haved changing comince of big data big box the base donner data haved changing by me and the districk haved changing by leathch noable of districk NoX PacT his comonshurheynal rouverange districk of the galhe of disturb by the reddick haved changing datta haved NAuthority and Authority of districk grand mendat lnternational apocalypsme of districk NoX-PAcT of the commende of the disturb data by NoX-PacT of the reverse districk aera lt's the reverse of distributing galhe of my dauther Andreinhexa

**Pull request merged**